### PR TITLE
added AWS configurations

### PR DESCRIPTION
**Pull request by arevikarakel**

### CODE REVIEW

File: .github/workflows/maven.yml [hunk 1]
```diff
@@ -26,5 +24,16 @@ jobs:
...
+        path: os-server/target/os-server-**.jar
+    - name: Rename file
+      run: mv os-server/target/os-server-**.jar os-server/target/os-server-${{ github.run_number }}.jar
+    - name: Upload to S3 as artifact
+      uses: medlypharmacy/s3-artifacts-action@master
+      with:
+        aws_access_key_id: ${{ secrets.AWS_S3_ACCESS_ID}}
+        aws_secret_access_key: ${{ secrets.AWS_S3_ACCESS_KEY}}
+        aws_s3_bucket_name: ${{ secrets.AWS_S3_BUCKET_NAME }}
+        source_path: 'os-server/target/os-server-${{ github.run_number }}.jar'
```
Comment (SevakMart) [inline .github/workflows/maven.yml:37]: source path will be different for you I think

File: .github/workflows/maven.yml [hunk 2]
```diff
@@ -26,5 +24,16 @@ jobs:
     - run: mkdir artifact && cp target/*.jar artifact
     - uses: actions/upload-artifact@v2
       with:
-       name: Package
-       path: artifact
+        name: os-server-${{ github.run_number }}.jar
```
Comment (SevakMart) [inline .github/workflows/maven.yml:27]: the jar name as well

File: .github/workflows/maven.yml (hunk 1 above)
Comment (arevikarakel) [reply .github/workflows/maven.yml]: The source path is changed

File: .github/workflows/maven.yml (hunk 2 above)
Comment (arevikarakel) [reply .github/workflows/maven.yml]: The jar name is also changed.

**Review by SevakMart — APPROVED** [review]

**Pull request merged**